src/Microsoft.OData.Cli/ProjectHelper.cs

```diff
@@ -0,0 +1,143 @@
+        {
+            ProjectCollection.GlobalProjectCollection.UnloadAllProjects();
+            Project project = CreateProjectInstance(projectPath);
+
+            return project;
+        }
+
+        /// <summary>
+        /// Adds a project item to the project
+        /// </summary>
+        /// <param name="project">An instance of the loaded <see cref="Project"/>/></param>
+        /// <param name="item">The project item to add to the project</param>
+        /// <param name="value">The value of the project item being added</param>
+        internal static void AddProjectItem(Project project, string item, string value)
+        {
+            if (project != null)
+            {
+                project.AddItem(item, value);
+                project.Save();
+            }
+        }
+
+        /// <summary>
+        /// This method determines whether the .sln file and .csproj file are in the same folder.
+        /// This is important in determining where to create the nuget packages folder for a project.
+        /// For a project that uses the packages folder structure.
+        /// </summary>
+        /// <param name="projectPath">A path to the directory with .csproj file</param>
+        /// <returns></returns>
+        internal static bool CheckIfSolutionAndProjectFilesAreInSameFolder(string projectPath)
+        {
+            string[] projectExtensions = { ".csproj", ".vbproj" };
+
+            string[] checkIfProjectExtensionExists = Directory.GetFiles(projectPath, "*.*")
+                .Where(f => projectExtensions.Contains(Path.GetExtension(f).ToLower())).ToArray();
+
+            string[] checkIfSlnExtensionExists = Directory.GetFiles(projectPath, "*.sln");
+
+            if (checkIfProjectExtensionExists.Length > 0 && checkIfSlnExtensionExists.Length > 0)
+            {
+                return true;
+            }
+            else 
+            {
+                return false;
+            }
+        }
+
+        /// <summary>
+        /// This method gets all the targetframeworks of the provided project. 
+        /// Limiting the package installation to only .net framework and .netcore targetframeworks.
+        /// </summary>
+        /// <param name="project"></param>
+        /// <returns></returns>
+        internal static string[] GetProjectTargetFrameworks(this Project project)
+        {
+            string[] targetFrameworks = new[] { "" };
+            string targetFramework = string.IsNullOrEmpty(project.GetPropertyValue("TargetFramework")) ? project.GetPropertyValue("TargetFrameworkVersion") : project.GetPropertyValue("TargetFramework");
+            if (string.IsNullOrEmpty(targetFramework) || targetFramework.Equals("v4.0"))
+            {
+                string multipleTargetFrameworks = project.GetPropertyValue("TargetFrameworks");
+                if (!string.IsNullOrEmpty(multipleTargetFrameworks))
+                {
+                    string[] frameworks = multipleTargetFrameworks.Trim().Split(';');
```

Comment:
```suggestion
                    IEnumerable<string> frameworks = multipleTargetFrameworks.Split(new[] { ';' }, StringSplitOptions.RemoveEmptyEntries).Select(d => d.Trim());
```
This should take care of scenarios like `<TargetFrameworks>net45; net472</TargetFrameworks>`